docs/docs/Release_Notes/2.1.0/2.1.0.md

```diff
@@ -0,0 +1,13 @@
+# Release - GA (v2.1.0)
+
+To install or upgrade the SDK, refer to instructions [here](/Getting_Started.md#installation).
+
+## New & Improved
+
+* Added the ability to migrate existing Lua toolkits to plugins.
+    * [Convert](/Versioning_And_Upgrade/Lua_Toolkit_To_SDK_Plugin_Migration/Converting_Migration_Scripts.md) any existing Lua upgrade scripts to Python migrations by using our new Lua upgrade [decorators](/Versioning_And_Upgrade/Lua_Toolkit_To_SDK_Plugin_Migration/Decorators.md).
+
+    More details about Lua Upgrade can be found [here](/Versioning_And_Upgrade/Lua_Toolkit_To_SDK_Plugin_Migration/Overview.md).
+
+* Added a new field `luaName` to the [Plugin Config](/Versioning_And_Upgrade/Lua_Toolkit_To_SDK_Plugin_Migration/Plugin_Config.md) which allows plugins to define the name of the LUA toolkit that the plugin can upgrade.
+* Added a new field `minimumLuaVersion` to the [Plugin Config](/Versioning_And_Upgrade/Lua_Toolkit_To_SDK_Plugin_Migration/Plugin_Config.md) which allows plugins to define the lowest LUA toolkit version that the plugin can upgrade.
```

Comment:
Do we want to leave a note somewhere that these 2 fields go together and cannot leave one or the other?

Reply:
Could maybe even just collapse together, something like:
* Added a new fields `luaName` and `minimumLuaVersion` to the [Plugin Config](/Versioning_And_Upgrade/Lua_Toolkit_To_SDK_Plugin_Migration/Plugin_Config.md). These allow the plugins to specify which Lua toolkit(s) that the plugin can upgrade from.

Reply:
I'll go with Tom's suggestion